panel/pane/markup.py

```diff
@@ -338,10 +339,11 @@ def applies(cls, obj, **params):
 
     def _get_properties(self):
         properties = super()._get_properties()
-        if isinstance(self.object, string_types):
-            text = self.object
-        else:
-            text = json.dumps(self.object or {}, cls=self.encoder)
+        try:
+            data = literal_eval(self.object)
```

Comment:
Should I use `json.loads` here instead of `literal_eval`?

Reply:
`json.loads` probably makes more sense.